Corona-Warn-App/src/main/res/values-uk/strings.xml

```diff
@@ -26,6 +26,8 @@
     <string name="menu_trash">"Кошик"</string>
     <!-- XMIT: social media -->
     <string name="menu_social_media">"Соціальна мережа"</string>
+    <!-- XMIT: Menu item accessibility content description -->
+    <string name="menu_item_accessibility_description">%s %d %d</string>
```

Comment:
it is enough to set `de` and default `en` only , translation team will take care of the rest

Reply:
[remove redundant menu_item_accessibility_description strings](https://github.com/corona-warn-app/cwa-app-android/pull/5464/commits/79402f7e313e88cd9ea799256041be84ac35767d)